src/actions/auth.js

```diff
@@ -104,7 +105,17 @@ export const loginWithFB = FBSDK => async (dispatch, getState) => {
         });
         await dispatch(loginWithToken(token));
       } catch (error) {
-        dispatch(toastNotificationAndRollbarAndThrowError('ER0005', error));
+        if (error instanceof GraphqlError) {
+          if (error && error.codes) {
```

Comment:
前面 instanceof 應該已經保證有了，同 L156

```suggestion
          if (error.codes) {
```

Reply:
我改了個實作方法了，please help review !